### PR TITLE
docs(appium): add environment variable docs

### PR DESCRIPTION
**Pull request by jlipps**
cc @appium/documentors

### CODE REVIEW

File: packages/appium/docs/en/cli/env-vars.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+variables, refer to the documentation for your operating system and terminal. These are the
+environment variables that the Appium server understands:
+
+|Variable&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Description|
+|--------|-----------|
+|`APPIUM_HOME`|By default, Appium creates a directory called `.appium` in the home directory for your system user. You can adjust the directory with this variable, as detailed in the [Managing Extensions](../guides/managing-exts.md) guide.|
+|`APPIUM_TMP_DIR`|By default, Appium uses a random temporary directory for many of its operations. If you wish to use a specific directory, you may do so by including an absolute path as the value of this variable. The behavrious is equivalent to using the `--tmp` CLI arg.|
+|`APPIUM_PREFER_SYSTEM_UNZIP`|Set to `1` to request that Appium use the `unzip` binary included on your system for unzipping downloaded apps or other artifacts, instead of the default JS-based unzip library.|
```
Comment (mykola-mokhnach) [inline packages/appium/docs/en/cli/env-vars.md:14]: actually, this is the default setting

Comment (jlipps) [reply packages/appium/docs/en/cli/env-vars.md]: ah ok! updated.

File: packages/appium/docs/en/cli/env-vars.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+environment variables that the Appium server understands:
+
+|Variable&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Description|
+|--------|-----------|
+|`APPIUM_HOME`|By default, Appium creates a directory called `.appium` in the home directory for your system user. You can adjust the directory with this variable, as detailed in the [Managing Extensions](../guides/managing-exts.md) guide.|
+|`APPIUM_TMP_DIR`|By default, Appium uses a random temporary directory for many of its operations. If you wish to use a specific directory, you may do so by including an absolute path as the value of this variable. The behavrious is equivalent to using the `--tmp` CLI arg.|
+|`APPIUM_PREFER_SYSTEM_UNZIP`|Set to `0` or `false` to request that Appium not use the `unzip` binary included on your system for unzipping downloaded apps or other artifacts. Instead it will use a JS-based unzip library. This could help on some systems with non-existent or non-standard `unzip` commands|
```
Comment (mykola-mokhnach) [inline packages/appium/docs/en/cli/env-vars.md:14]: actually the code will fallback to the nodejs's built-in extractor anyway. they problem with the latter is that it does not perform as well as the system one (which is written in C and uses multithreading properly)

File: packages/appium/docs/en/cli/env-vars.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Server Environment Variables
+---
+
+The primary way of configuring the Appium server is via [CLI Args](./args.md). However, some more
+advanced features are toggled or configured via environment variables. To set environment
+variables, refer to the documentation for your operating system and terminal. These are the
+environment variables that the Appium server understands:
+
+|Variable&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Description|
```
Comment (eglitise) [inline packages/appium/docs/en/cli/env-vars.md:10]: Are these non-breaking spaces needed? Other doc files don't seem to have them...

File: packages/appium/docs/en/cli/env-vars.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+|--------|-----------|
+|`APPIUM_HOME`|By default, Appium creates a directory called `.appium` in the home directory for your system user. You can adjust the directory with this variable, as detailed in the [Managing Extensions](../guides/managing-exts.md) guide.|
+|`APPIUM_TMP_DIR`|By default, Appium uses a random temporary directory for many of its operations. If you wish to use a specific directory, you may do so by including an absolute path as the value of this variable. The behavrious is equivalent to using the `--tmp` CLI arg.|
```
Comment (eglitise) [inline packages/appium/docs/en/cli/env-vars.md:13]: Small typo

File: packages/appium/docs/en/cli/env-vars.md (hunk 3 above)
Comment (jlipps) [reply packages/appium/docs/en/cli/env-vars.md:10]: without them, the env vars are wrapped mid-token

**Review by eglitise — APPROVED** [review]

**Pull request merged**